### DISCUSSION TITLE
I want to use the liver dataset for training

### DISCUSSION THREAD

**Discussion by zzx332** [voxelmorph/voxelmorph#166] [Q&A] [answered]
1.If I want to use the liver dataset for training，how should I choose the image pair（moving image and fixed image）？
2.Are there any requirements between these two images，or choose two images directly at random？
3.How many images do i need？

**Comment by adalca** [comment, marked as answer]
@zzx332 

1. I'm not sure which dataset you mean, but if you're interested in aligning two liver images to each other I would just randomly choose a pair of images
2. not really, random is ok.
3. It depends, and we did some analysis of the number of needed images here: https://arxiv.org/abs/1809.05231 . If you use some data augmentation (intensity and spatial transforms), I suspect you can get away with 20-50 images.
